cypress/integration/basic_infinite_spec.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```

Comment:
Which eslint rules does this break? If it's just the `cy` variable, you could drop a `.eslintrc` config in the cypress directory with `cy` as a global